Editor/MobiledgeXEditorWindow.cs

```diff
@@ -330,24 +330,21 @@ public async Task<bool> CheckCredentials()
                     Debug.LogError("No Mapped Ports for your application backend");
                 }
                 return checkResult;
-
-
-            }
-            catch (HttpException httpe) // HTTP status, and REST API call error codes.
-            {
-                // server error code, and human readable message:
-                clog("MobiledgeX: RegisterClient Exception ", httpe.Message + ", HTTP StatusCode: " + httpe.HttpStatusCode + ", API ErrorCode: " + httpe.ErrorCode + "\nStack: " + httpe.StackTrace, true);
-                return false;
             }
             catch (HttpRequestException httpre)
             {
                 clog("MobiledgeX: RegisterClient HttpRequest Exception", httpre.Message + "\nStack Trace: " + httpre.StackTrace, true);
                 return false;
             }
-            catch (FindCloudletException findCloudletException)
+            catch (RegisterClientException rce) 
+			{
```

Comment:
Nit: whitepace weird.

Reply:
RegisterClientException. RegisterClient() catches HttpException and then throws a RegisterClientException with status, message, error code